apps/web/views.py

```diff
@@ -41,42 +41,38 @@ def internal_server_error(request):
     response.status_code = 500
     return response
 
-
+@permission_classes((permissions.IsAuthenticated,))
```

Comment:
@charanjit-singh Using this decorator doesn't solve our purpose since this API can only be accessed by `superuser`.